tests/expectations/metrics/test_map_metric.py

```diff
@@ -370,12 +370,22 @@ def test_get_map_metric_dependencies():
 
 
 def test_is_sqlalchemy_metric_selectable():
-    assert MapMetricProvider.is_sqlalchemy_metric_selectable(
-        map_metric_provider=CompoundColumnsUnique
+    with pytest.warns(DeprecationWarning) as record:
+        assert MapMetricProvider.is_sqlalchemy_metric_selectable(
+            map_metric_provider=CompoundColumnsUnique
+        )
+
+    assert "MapMetricProvider.is_sqlalchemy_metric_selectable is deprecated." in str(
```

Comment:
@abegong Non-blocking: I need to learn more about what led to the `MapMetricProvider.is_sqlalchemy_metric_selectable()` method becoming deprecated.

Adding @anthonyburdi and @Shinnnyshinshin who might be able to offer more detail regarding this deprecation.  

Thank you.